setup.py

```diff
@@ -22,7 +22,8 @@
     "pyyaml",
     "cloudpickle>=1.6.0,<=2.2.1,!=2.1.0,!=2.2.0;python_version<'3.11'",
     "cloudpickle==2.2.1;python_version~='3.11'",  # backend only supports cloudpickle 2.2.1 + python 3.11 at the moment
-    "protobuf>=5.28",  # Snowpark IR
+    # `protoc` < 3.20 is not able to generate protobuf code compatible with protobuf >= 3.20.
+    "protobuf>=3.20, <6",  # Snowpark IR
```

Comment:
Consider dropping the upper bound. Since so many other Snowflake dependencies specify it, we don't really need it. And we don't need it for ourselves, either.

Reply:
I'd like to keep it consistent with `Streamlit`. Whether to drop upper bound should be a broader discussion across all company wise Python repos.